### PR TITLE
Change remote method as isolated remote method

### PR DESCRIPTION
**Pull request by Chuhaa**
## Purpose
Change remote methods as isolated remote methods to support S3 connector methods inside users' isolated methods.

## Goals
Fix https://github.com/ballerina-platform/ballerina-extended-library/issues/18

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
SL Beta3

### CODE REVIEW

**Comment by LakshanSS** [comment]
Looks good to me.

Apart from the function isolation, we have done some other minor improvements here.
Shall we also mention about those changes in the PR description?

**Review by LakshanSS — APPROVED** [review]

**Comment by Chuhaa** [comment]
> Looks good to me.
> 
> Apart from the function isolation, we have done some other minor improvements here.
> Shall we also mention about those changes in the PR description?

Yes. It is better to add them in another PR.

File: s3/sort.bal [hunk 1]
```diff
@@ -14,24 +14,68 @@
...
-"t", "u", "v", "w", "x", "y", "z"];
 const string NON_ALPHABET = "nonAlp";
 
 class SortBucket {
-
+    private final string[] alphabet = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", 
+"t", "u", "v", "w", "x", "y", "z"];
     string[] items = [];
     int index = 0;
     string[] result = [];
```
Comment (abeykoon) [inline s3/sort.bal:24]: Shall we make other class variables also `private`?

Comment (Chuhaa) [reply s3/sort.bal]: Fixed in https://github.com/ballerina-platform/module-ballerinax-aws.s3/commit/f165d4bb6675e836e59f71c1a2a09a5ebe95c1c8

**Review by abeykoon — APPROVED** [review]

**Pull request merged**